### DISCUSSION TITLE
Windows Store

### DISCUSSION THREAD

**Discussion by abcpop** [valinet/ExplorerPatcher#474] [General]
Are those update related to EP ?
It reappears after a restart of windows

[<a href="https://ibb.co/NKrvs9H"><img src="https://i.ibb.co/FzXrK5j/Image1.jpg" alt="Image1" border="0"></a>](url)

**Comment by valinet** [comment]
No. EP is as far away as a Windows app can be from a UWP =))))

**Reply by abcpop** [reply]
Thanks
